Makefile

```diff
@@ -121,15 +121,17 @@ endif
 ENABLE_JIT ?= 0
 $(call set-feature, JIT)
 ifeq ($(call has, JIT), 1)
-OBJS_EXT += jit_x64.o 
+OBJS_EXT += jit.o 
 ifneq ($(processor), x86_64)
-$(error JIT mode only supports for x64 target currently.)
+ifneq ($(processor), aarch64)
```

Comment:
Use `filter` instead.
See https://www.gnu.org/software/make/manual/html_node/Text-Functions.html

Reply:
The result of `uname -m` on macOS/Arm64 is `arm64`.